README.md

```diff
@@ -109,8 +109,30 @@ Transport
 
 py_zipkin (for the moment) thrift-encodes spans. The actual transport layer is
 pluggable, though. The `transport_handler` is a function that takes a single
-argument - the thrift-encoded bytes. An example of sending spans to Kafka
-using the [kafka-python](https://pypi.python.org/pypi/kafka-python) package:
+argument - the thrift-encoded bytes.
+
+The simplest way to get spans to the collector is via HTTP POST. Here's an
+example of a simple HTTP transport using the `requests` library. This assumes
+your Zipkin collector is running at localhost:9411.
+
+```python
+import requests
+
+def http_transport(message):
```

Comment:
the transport handler is supposed to be a function, so try passing in http_transport as the parameter value instead.

transport_handler=http_transport,